README.md

```diff
@@ -16,3 +16,17 @@ Also give a theoretical argument for why X could or could not be correct, based
 on the complexity of the general sorting problem we covered in class.
 
 Add your answers to this markdown file.
+
+## My Approach
+The only way to test a function like this is to approach it from many different angles. Any sorting algorithm should, in theory, be able to sort any list given to it. The only way to ensure it is indeed better than the competition is to compare its runtimes for any given style of list to every other known general sorting algorithm.
+
+The fastest way I can think of to test this, is to feed the program a series of lists that matches each other general sorting algorithm's worst case scenario and compare the performances of every general sorting algorithm and the new algorithm.
+
+For example, competing against insertion sort, we would compare the new algorithms ability to process a reverse sorted list, i.e. [10,9,8,7...,3,2,1]
+Insertion sort handles this at a time of $O(n^2)$, if the new algorithm can process this list, and other lists following the same pattern of varying lengths and complexities, then the algorithm can be said to outperform insertion sort. This exact method, applied to every other general sorting algorithm is how we would prove that this new algorithm was superior to every general sorting algorithm.
```

Comment:
What would be your theoretical argument for why X could or could not be correct? Based on the complexity of the general sorting problem we covered in class.

Reply:
Updated with my reasoning